docs/INSTALL.md

```diff
@@ -11,27 +11,67 @@ In order to install the *job-executor-service* on the remote execution plane, so
+helm upgrade --install --create-namespace -n <NAMESPACE> job-executor-service \
+    https://github.com/keptn-contrib/job-executor-service/releases/download/${JES_VERSION}/job-executor-service-${JES_VERSION}.tgz \
+    --set remoteControlPlane.topicSubscription=${TASK_SUBSCRIPTION} \
+    --set remoteControlPlane.api.protocol=${KEPTN_API_PROTOCOL} \
+    --set remoteControlPlane.api.hostname=${KEPTN_API_HOST} \
+    --set remoteControlPlane.api.token=${KEPTN_API_TOKEN}
 ```
 
 Please replace `<VERSION>` with the actual version you want to install from the compatibility matrix above or the
 [GitHub releases page](https://github.com/keptn-contrib/job-executor-service/releases).
 
 To verify that everything works you can visit Bridge, select a project, go to Uniform, and verify that `job-executor-service`  is registered as "remote execution plane" with the correct version and event type.
 
-**Example (with auto-detection)**
+**Example installation with OAuth:**
+
+If the oauth authentication mode should be used instead of the token, the following properties must be specified in addition to the setting `remoteControlPlane.api.authMode` to `oauth`:
+* `remoteControlPlane.api.oauth.clientId` - The OAuth client ID
+* `remoteControlPlane.api.oauth.clientSecret` - The OAuth client secret
+* `remoteControlPlane.api.oauth.clientDiscovery` - The well known discovery URL for the OAuth endpoint (e.g.: `https://your.sso.domain/.well-known/openid-configuration`)
+* `remoteControlPlane.api.oauth.scopes` - The scopes which should be requested by the job executor service
+
+```bash
+JES_VERSION=<VERSION> # e.g. 0.2.1
+
+KEPTN_API_PROTOCOL=http # or https
+KEPTN_API_HOST=<INSERT-YOUR-HOSTNAME-HERE> # e.g., 1.2.3.4.nip.io
+
+KEPTN_OAUTH_CLIENT_ID=<INSERT-YOUR-CLIENT_ID>
+ KEPTN_OAUTH_CLIENT_SECRET=<INSERT-YOUR-CLIENT_SECRET>
+KEPTN_OAUTH_DISCOVERY=<INSERT-YOUR-DISCOVERY_URL>
+KEPTN_OAUTH_SCOPES=<INSERT-YOUR-OAUTH-SCOPES>
+
+TASK_SUBSCRIPTION=sh.keptn.event.remote-task.triggered
+
+helm upgrade --install --create-namespace -n <NAMESPACE> job-executor-service \
+    https://github.com/keptn-contrib/job-executor-service/releases/download/${JES_VERSION}/job-executor-service-${JES_VERSION}.tgz \
+    --set remoteControlPlane.autoDetect.enabled="false" \
+    --set remoteControlPlane.topicSubscription="${TASK_SUBSCRIPTION}" \
+    --set remoteControlPlane.api.protocol=${KEPTN_API_PROTOCOL} \
+    --set remoteControlPlane.api.hostname=${KEPTN_API_HOST} \
+    --set remoteControlPlane.api.authMode="oauth" \
+    --set remoteControlPlane.api.oauth.clientId=${KEPTN_OAUTH_CLIENT_ID} \
+    --set remoteControlPlane.api.oauth.clientSecret=${KEPTN_OAUTH_CLIENT_SECRET} \
+    --set remoteControlPlane.api.oauth.clientDiscovery=${KEPTN_OAUTH_DISCOVERY} \
+    --set remoteControlPlane.api.oauth.scopes=${KEPTN_OAUTH_SCOPES}
+```
+
+**Example installation with auto-detection:**
```

Comment:
I haven't tested auto-detection with OAuth, but I don't think that it does make sense, because for the OAuth flow you have to specify a bunch of parameters (client id, secret, scopes, ...) which should be unique to the keptn instance that is used. The auto-detection is primarily here to simplify the connection to a keptn instance that is in the same cluster as JES.